mathics/core/expression.py

```diff
@@ -1382,14 +1382,26 @@ def __getnewargs__(self):
         return (self._head, self._leaves)
 
 
+# This function allows to avoid a circular reference when we want
```

Comment:
Change to: 

This avoids a circular imports by allowing creation of  `Expression` objects in `mathics.core.symbols` and `mathics.core.atoms` modules.